HW02/Lizaveta_Muromskaya/spec/my_array_methods_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative 'my_array_methods'
+using MyArrayMethods
+
+RSpec.describe 'MyArrayMethods' do
+  empty_array = []
+  array = [1, 2, 3, 4, 5]
+
+  describe '.my_each' do
+    context 'when block is given' do
+      context 'when array is empty' do
+        it 'return nothing' do
+          expect(empty_array.my_each {}).to eq(0..-1)
+        end
+      end
```

Comment:
Instead of adding a new variable would be better to rewrite the previous one 
```suggestion
    context 'when block is given' do
      context 'when array is empty' do
        let(:array) { [] }
        
        it 'return nothing' do
          expect(array.my_each {}).to eq(0..-1)
        end
      end
```